### PR TITLE
refactor: Update tutorials

### PR DESCRIPTION
**Pull request by samricotta**
Setting up a preliminary tutorials tab to link to individual tutorials to be able to insert new tutorials in easily

Part of #84

### CODE REVIEW

**Comment by julienrbrt** [comment]
Could you post a screenshot?

**Comment by samricotta** [comment]
Screenshot 2023-11-21 at 14 05 46

File: sidebars.js [hunk 1]
```diff
@@ -14,9 +14,79 @@
...
+      items: [
+        {
+          type: "link",
+          label: "Run a Node, API, and CLI",
+          href: "https://tutorials.cosmos.network/tutorials/3-run-node/",
+        },
+        {
+          type: "link",
+          label: "Run a Cosmos Chain with Ignite CLI",
+          href: "https://tutorials.cosmos.network/hands-on-exercise/1-ignite-cli/",
```
Comment (julienrbrt) [inline sidebars.js:80]: ditto

File: sidebars.js [hunk 2]
```diff
@@ -14,9 +14,79 @@
...
+            items: [
+        {
+          type: "link",
+          label: "Run a Node, API, and CLI",
+          href: "https://tutorials.cosmos.network/tutorials/3-run-node/",
+        },
+        {
+          type: "link",
+          label: "Run a Cosmos Chain with Ignite CLI",
+          href: "https://tutorials.cosmos.network/hands-on-exercise/1-ignite-cli/",
```
Comment (julienrbrt) [inline sidebars.js:55]: ditto

**Comment by julienrbrt** [comment]
Why does it need to be repeated 3 times? 🤔

File: sidebars.js [hunk 3]
```diff
@@ -14,9 +14,79 @@
...
```
[the remark this answers is not in the document]
Comment (samricotta) [reply sidebars.js:31]: hmmm, this is true. I will wait then and just include my changes for the tutorials

**Comment by samricotta** [comment]
Repeated so that it lives on all pages, probably should just live on learn but I dont want it to get lost in there. Eventually I think having it at the top as a tab when we have more content @julienrbrt

**Pull request closed without merging**